web/themes/joinup/joinup_theme.theme

```diff
@@ -1,6 +1,42 @@
 <?php
-
 /**
  * @file
  * Functions to support theming in the Joinup theme.
  */
+
+use Drupal\image\Entity\ImageStyle;
+use \Drupal\file\Entity\File;
+
+/**
+ * Implements hook_preprocess_page().
+ */
+function joinup_theme_preprocess_page(&$variables) {
+  // Set page title.
```

Comment:
Is the title really not available to the variables of the page by default? I mean, really? we need to provide this?